### PR TITLE
tidy up new fltables implementation

### PR DESCRIPTION
**Pull request by WalterBright**
mostly whitespace changes

### CODE REVIEW

File: src/dmd/backend/cg.d [hunk 1]
```diff
@@ -49,111 +51,112 @@ Symbol *tls_get_addr_sym;       // function __tls_get_addr
...
+                   FLndp,
+                 ])
+    {
+        stackfl[fl] = true;
+    }
+    return stackfl;
+} ();
+
+/// What segment register is associated with it?
+ubyte[FLMAX] segfl =
```
Comment (Geod24) [inline src/dmd/backend/cg.d]: `immutable` ?

File: src/dmd/backend/cg.d [hunk 2]
```diff
@@ -49,111 +51,112 @@ Symbol *tls_get_addr_sym;       // function __tls_get_addr
...
-        return datafl;
-    } ();
-
+    }
+
+    return segfl;
+} ();
+
+/// Is fl in the symbol table?
+bool[FLMAX] flinsymtab =
```
Comment (Geod24) [inline src/dmd/backend/cg.d]: `immutable`?

**Review by Geod24 — COMMENTED** [review]
Just a few nits.

It also looks like you could make https://github.com/dlang/dmd/blob/23c422403c41d88c33c88551c1ee969d6a900310/src/dmd/backend/cc.d#L1605 a "proper" `enum` (e.g. `enum FL`) and replace `FLMAX` with `FL.max` in a follow up PR.

File: src/dmd/backend/cg.d (hunk 2 above)
Comment (WalterBright) [reply src/dmd/backend/cg.d]: immutable would require more refactoring, as it is not C++ compatible. So not this time.

Comment (Geod24) [reply src/dmd/backend/cg.d]: What needs to be compatible with C++ ? Is it for DMD, or DMC ?

Comment (Geod24) [reply src/dmd/backend/cg.d]: Also this is removing `C++` linkage from those data structure so they are not "C++ compatible" anymore

Comment (WalterBright) [reply src/dmd/backend/cg.d]: no, it isn't. Look at the whole file.

Comment (WalterBright) [reply src/dmd/backend/cg.d]: > What needs to be compatible with C++ ?

D mangles the name to include the module name. C++ does not. Therefore, switching to D linkage means the modules need to be reorganized. This is not the PR to do that.

File: src/dmd/backend/cg.d [hunk 3]
```diff
@@ -49,111 +51,112 @@ Symbol *tls_get_addr_sym;       // function __tls_get_addr
...
-{
-    ubyte[FLMAX] datafl =
-    () {
-        ubyte[FLMAX] datafl;
-        foreach (fl; [ FLdata,FLudata,FLreg,FLpseudo,FLauto,FLfast,FLpara,FLextern,
-                       FLcs,FLfltreg,FLallocatmp,FLdatseg,FLtlsdata,FLbprel,
-                       FLstack,FLregsave,FLfuncarg,
-                       FLndp, FLfardata,
-                     ])
+/// Is fl data?
```
[the remark this answers is not in the document]
Comment (WalterBright) [reply src/dmd/backend/cg.d]: See cc.d

**Review by Geod24 — APPROVED** [review]

**Pull request merged**